modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -46,6 +47,34 @@ sda.enkf.multisite <- function(settings,
   ###-------------------------------------------------------------------###
   ### read settings                                                     ###
   ###-------------------------------------------------------------------###
+  
+  ### if user wants a different outdir than /SDA, edit the xml to output where you want it.
+  if (!(is.null(outfolder)))
```

Comment:
`outfolder` is not defined before and it's argument either, it think it may break the code.